redisinsight/ui/src/pages/browser/modules/key-details/components/list-details/remove-list-elements/RemoveListElements.tsx

```diff
@@ -261,24 +261,30 @@ const RemoveListElements = (props: Props) => {
                 />
               </FormField>
             </FlexItem>
-            <FlexItem grow style={{ width: '100%' }}>
-              <FormField
-                additionalText={!canRemoveMultiple ? InfoBoxPopover() : <></>}
-              >
-                <TextInput
-                  name={config.count.name}
-                  id={config.count.name}
-                  maxLength={200}
-                  placeholder={config.count.placeholder}
-                  value={count}
-                  data-testid="count-input"
-                  autoComplete="off"
-                  onChange={handleCountChange}
-                  ref={countInput}
-                  disabled={!canRemoveMultiple}
-                />
-              </FormField>
-            </FlexItem>
+            <Row grow>
+              <FlexItem grow>
+                <FormField>
+                  <TextInput
+                    name={config.count.name}
+                    id={config.count.name}
+                    maxLength={200}
+                    placeholder={config.count.placeholder}
+                    value={count}
+                    data-testid="count-input"
+                    autoComplete="off"
+                    onChange={handleCountChange}
+                    ref={countInput}
+                    disabled={!canRemoveMultiple}
+                  />
+                </FormField>
+              </FlexItem>
+
+              {!canRemoveMultiple ? (
```

Comment:
Bonus Points: Not related to the changes of the popover colors, but I fixed this minimal spacing issue as well, so it will look like in the old version, before the EUI replacement.

| Old | Before | After |
| - | - | - |
<img width="1303" height="821" alt="Image" src="https://github.com/user-attachments/assets/868bbaca-cdea-4b99-8387-7cf329dc0a88" />|<img width="1173" height="798" alt="Image" src="https://github.com/user-attachments/assets/536aa1f5-0e43-4ac2-82eb-61b1129a0c68" />|<img width="1100" height="658" alt="Image" src="https://github.com/user-attachments/assets/ac014c46-55a3-4731-a80a-7c40a11c644a" />